conf/prod.routes

```diff
@@ -3,5 +3,6 @@
 ->         /account              app.Routes
 ->         /                     health.Routes
 ->         /template             template.Routes
+->         /internal             internal.Routes
```

Comment:
I had some problems with routing when I tried adding a `.Route` file to a stub, so please check this works ok on the dev environment